### PR TITLE
update validator doc for v0.8.0 and laconic_81337-6

### PR DESCRIPTION
**Pull request by iszubok**

### CODE REVIEW

File: testnet/genesis-validators.md [hunk 1]
```diff
@@ -271,25 +272,33 @@ journalctl -f -u laconicd
 In this example the Tendermint RPC and Prometheus metrics ports are exposed only to localhost. You may want to change 127.0.0.1 to private or public network interface of your host if you need to access these ports remotely.
 
 ```sh
-docker create --name laconic-testnet-5 -v ~/.laconicd:/root/.laconicd -p 26656:26656 -p 127.0.0.1:26657:26657 -p 127.0.0.1:26660:26660 git.vdb.to/cerc-io/laconicd/laconicd:v0.6.0 laconicd start --gql-playground --gql-server --log_level=warn
+docker create \
```
Comment (i-norden) [inline testnet/genesis-validators.md:275]: Much easier to read 👍

File: testnet/genesis-validators.md [hunk 2]
```diff
@@ -122,13 +123,13 @@ Make sure the directory `~/.laconicd` does not exist or is empty
 >In order to run the below commands in a docker container:
 >```sh
 >docker run -ti -v ~/.laconicd:/root/.laconicd \
->git.vdb.to/cerc-io/laconicd/laconicd:v0.6.0 /bin/sh
+>git.vdb.to/cerc-io/laconicd/laconicd:v0.7.0 /bin/sh
```
Comment (i-norden) [inline testnet/genesis-validators.md:126]:
```suggestion
>git.vdb.to/cerc-io/laconicd/laconicd:v0.8.0 /bin/sh
```

**Review by i-norden — CHANGES_REQUESTED** [review]
Just one small edit, otherwise LGTM. Thanks!

**Comment by srwadleigh** [comment]
looks good to me, thanks!

**Review by srwadleigh — APPROVED** [review]

File: testnet/genesis-validators.md (hunk 2 above)
Comment (iszubok) [reply testnet/genesis-validators.md:126]: Good catch. Thank you!

**Pull request merged**